### ISSUE TITLE
Some error in predict~~

### ISSUE THREAD

**Issue by WFLiu0327** [FengheTan9/Medical-Image-Segmentation-Benchmarks#3]
We are getting error in predicting single images after training these models, the predicted images are black, can you please upload the script for predicting single images as well, thanks a lot!

**Comment by FengheTan9** [comment]
`mask = output.cpu().detach().numpy()[0][0] * 255`
